modules/ocf_kubernetes/manifests/master/loadbalancer/ssl.pp

```diff
@@ -0,0 +1,20 @@
+class ocf_kubernetes::master::loadbalancer::ssl(
+  $server_name,
+) {
+
+  $cnames = ldap_attr_multi($server_name, 'dnsCname')
+
+  $cnames.each |String $alias| {
+    $service_fqdn = "${alias}.ocf.berkeley.edu"
+    ocf::ssl::bundle { $service_fqdn:
+      domains => [$service_fqdn, "${alias}.ocf.io"],
```

Comment:
does this not get handled for you by the default bundle anyways? IMO it would be preferable to not hard code .ocf.berkeley.edu or .ocf.io in a deep child class, though it's unlikely to ever change, it seems like something that should be configured at a higher level of abstraction.